packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect, HTMLAttributes, ReactNode } from "react"
+import { v4 } from "uuid"
+import classnames from "classnames"
+import { history } from "prosemirror-history"
+import { keymap } from "prosemirror-keymap"
+import { EditorState } from "prosemirror-state"
+import { Node } from "prosemirror-model"
+import { Label } from "@kaizen/draft-form"
+import { baseKeymap } from "prosemirror-commands"
+import { buildKeymap, useRichTextEditor } from "@cultureamp/rich-text-toolkit"
+import { OverrideClassName } from "@kaizen/component-base"
+import { EditorContentArray } from "./types"
+import { createSchemaFromControls } from "./schema"
+import styles from "./RichTextEditor.scss"
+
+export interface RichTextEditorProps
+  extends OverrideClassName<Omit<HTMLAttributes<HTMLDivElement>, "onChange">> {
+  onChange: (content: EditorContentArray) => void
+  value: EditorContentArray
+  labelText: ReactNode
+}
+
+export const RichTextEditor: React.VFC<RichTextEditorProps> = props => {
+  const { onChange, value, labelText, classNameOverride, ...restProps } = props
+  const labelId = v4()
```

Comment:
This should use a `useMemo` or similar, otherwise it'll rerun and create a new ID on each render.

Reply:
Nice catch

Reply:
I did a quick search regarding `useMemo` vs `useState` for this and opted for `useState` after reading this https://stackoverflow.com/questions/57664450/usememo-vs-usestate-for-react-hooks-constants